API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -32,6 +32,53 @@ GeoJSON does not cover all use cases. For example, it is not possible to store c
 - When supporting GML, do this according to OGC API Features [Requirements class 8.4](https://docs.ogc.org/is/17-069r3/17-069r3.html#_requirements_class_geography_markup_language_gml_simple_features_profile_level_0) for GML Simple Features level 0, or [Requirements class 8.4](https://docs.ogc.org/is/17-069r3/17-069r3.html#_requirements_class_geography_markup_language_gml_simple_features_profile_level_2) for GML Simple Features level 2. 
 - Use a workaround, e.g. convert circular lines / arcs to regular linestrings. 
 
+Example of embedding WKT in a JSON object:
+<pre>
+// definition of JSON object
+//
+// type: object
+// required:
+//   - building
+// properties:
+//   building:
+//     type: object
+//     required:
+//       - geometry
+//     properties:
+//       geometry:
+//         type: string
+//         format: wkt
+{
+  "building": {
+    "geometry": "POLYGON Z((194174.445 465873.676 0, 194174.452 465872.291 0, 194158.154 465872.213 0, 194158.226 465856.695 0, 194223.89 465856.969 0, 194223.821 465872.48 0, 194207.529 465872.415 0, 194207.505 465882.528 0, 194207.498 465883.902 0, 194223.799 465883.967 0, 194223.732 465899.48 0, 194216.55 465899.45 0, 194215.15 465899.445 0, 194213.85 465899.439 0, 194158.068 465899.211 0, 194158.148 465883.685 0, 194174.42 465883.767 0, 194174.445 465873.676 0))"
+  }
+}
+</pre>
+
+Example of embedding GML as a base64 string in a JSON object:
+<pre>
+// definition of JSON object
+//
+// type: object
+// required:
+//   - terrain
+// properties:
+//   terrain:
+//     type: object
+//     required:
+//       - geometry
+//     properties:
+//       geometry:
+//         type: string
+//         contentType: application/gml+xml; charset=utf-8; version=3.2
```

Comment:
Regels 73 en 74 suggereren dat deze gegevens onderdeel uitmaken van het JSON object, maar deze worden via headers teruggegeven. Deze headers gelden voor de hele response. De response mag dan geen JSON bevatten met daarin alleen de geometrie als GML, de hele response moet dan in GML/XML zijn.

Reply:
Over contentType en contentEncoding was ik niet zeker, de regels zijn weggehaald.

Ik kan geen bestaande definitie voor `format: gml` kunnen vinden. Voor nu heb ik `format: byte` gekozen, omdat OpenAPI 3.0 `type: string` + 'format: byte` heeft gedefinieerd voor base64 karakters. Helaas wordt hiermee niet aangegeven dat de inhoud van de base64 tekst valide GML moet zijn.

Reply:
George gaat het embedded GML voorbeeld verwijderen. Daarna kan dit PR gemerged worden.